src/codeu/chat/client/core/Controller.java

```diff
@@ -28,9 +28,12 @@
 import codeu.chat.util.Uuid;
 import codeu.chat.util.connections.Connection;
 import codeu.chat.util.connections.ConnectionSource;
+import java.util.ArrayList;
 
 final class Controller implements BasicController {
 
+  public final ArrayList<Uuid> members = new ArrayList<>();
```

Comment:
This should be a member of the ConversationHeader here:

https://github.com/lianasyrkett/CodeU-Summer-2017/blob/master/src/codeu/chat/common/ConversationHeader.java#L56

You would then end up doing something like `conversation.members.add(member)` in the `addMemberToConvo` function below.